### ISSUE TITLE
In classes using typing generics, it seems parsing parent classes does not work

### ISSUE THREAD

**Issue by mitar** [davidhalter/jedi#1012]
See the following example:

```python
b = """
import typing

T = typing.TypeVar('T')

class Foo(typing.Generic[T]):
    def __init__(self, foo: T) -> None:
        self.foo = foo

class Bar(Foo[int]):
    def __init__(self, foo: int, bar: typing.Any) -> None:
        super().__init__(foo=foo)

        self.bar = bar

b = Bar()
b."""

completions = jedi.api.Script(source=b).completions()

print(completions)
```

I would expect that among completions there are both `bar` and `foo`, but it is just `bar`.

If I remove `typing.Generic` from the base class and corresponding code, then it works.

**Comment by hamologist** [comment]
Experiencing the same behavior as well.

**Comment by davidhalter** [comment]
It looks like this was fixed in 0.14.0 a month ago. (along with stub files and a lot of other things). Thanks for the report and have fun!
